src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -2929,6 +2929,8 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
 		this._baseCellEditorOptions.forEach(v => v.dispose());
 		this._baseCellEditorOptions.clear();
 
+		this._notebookOverviewRulerContainer.remove();
```

Comment:
Thanks for handling this! This part I don't think I follow. This is holding onto the actionbar somehow? And then the actionbar context holds onto the model?

Reply:
Yeah, I don't have a sound explanation tbh. I saw the retainer path in https://github.com/microsoft/vscode/issues/167813#issue-1470915064 and decided to remove the node for good measure